packages/suite/src/components/guide/GuideDefault.tsx

```diff
@@ -79,26 +80,22 @@ export const GuideDefault = () => {
             <GuideHeader label={<Translation id="TR_GUIDE_VIEW_HEADLINE_LEARN_AND_DISCOVER" />} />
             <GuideContent>
                 <GuideSearch pageRoot={indexNode} setSearchActive={setSearchActive} />
-                {!searchActive && (
-                    <GuideCategories
-                        node={indexNode}
-                        label={<Translation id="TR_GUIDE_CATEGORIES" />}
-                    />
-                )}
+                {!searchActive && <GuideCategories node={indexNode} />}
             </GuideContent>
+            <FeedbackBorder />
```

Comment:
nit: if padding is in the wrapper then this can simply be a border-top :)

Reply:
It can't be because it's not full-width line anymore: 
<img width="374" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/f62736bf-165b-4edd-9bd2-6741a6aa39ae">

I could implement it with a combination of paddings and margins, but this seems to me more in the case of readability.

Reply:
i know that it's not full width, I meant if the whole wrapper had the padding then the border would look like it does on the screenshot :)